index.mjs

```diff
@@ -0,0 +1,130 @@
+//importing game functions and images from resources folder
+import { ask, inputSanitizer, isInteger, random } from './resources/usableFunctions.mjs'
+import { title, win } from './resources/ascii_images.mjs'
```

Comment:
Good organization of your code, separating helper functions from functions containing all of your logic